draft-ietf-quic-invariants.md

```diff
@@ -178,16 +191,17 @@ version-specific semantics are marked with an X.
 
 A QUIC packet with a short header has the high bit of the first octet set to 0.
 
-A QUIC packet with a short header includes an optional connection ID and no
-version field.  The second bit of that octet (that is, 0x40) determines whether
-the connection ID is present.  If the second bit is cleared, a 64-bit connection
-ID immediately follows the first octet.  If the second bit is set, the remainder
-of the packet has version-specific semantics.
+A QUIC packet with a short header includes a Destination Connection ID.  The
```

Comment:
Just double checking: The short header does actually not include a connection ID if the length was zero in the long header, right?

Reply:
Yep, either that or it includes a zero-length connection ID :)